test/features/pagecontent/language_variants.js

```diff
@@ -158,7 +158,7 @@ describe('Language variants', function() {
             assert.deepEqual(res.headers['content-language'], 'de');
             assert.deepEqual(res.headers['x-restbase-sunset'] || null, 'true');
             assert.checkString(res.headers.etag, /^"\d+\/[a-f0-9-]+"$/);
-            assert.deepEqual('Das ist eine testseite', res.body.extract);
+            assert.deepEqual('Das ist eine testseite.', res.body.extract);
```

Comment:
The content of the page on beta was actually changed! Took me forever to figure that out...

https://de.wikipedia.beta.wmflabs.org/w/index.php?title=RESTBase_Testing_Page&diff=prev&oldid=30418